smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -48,9 +50,22 @@ def setUp(self):
                 if not '.' in tmp:
                     self._members.append(tmp)
 
+            self.session.set(['interfaces','dummy','dum10001'])
```

Comment:
You continously do not follow the code path. I requested to move this out of the general interface setup for multiple times now.